migration/mule/exposing-a-restful-resource-using-the-http-connector/ExposeRESTfulService/src/main/synapse-config/api/RESTfulService.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/person" name="RESTfulService" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST">
+        <inSequence>
+            <payloadFactory description="Mock payload data saved " media-type="json">
+                <format>
+					{
+					"personId":"1",
+					"message":"Person was created"
+					}
+				</format>
+                <args/>
+            </payloadFactory>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="GET" uri-template="/{personId}">
+        <inSequence>
+            <payloadFactory description="Mock Payload Retrieve one " media-type="json">
+                <format>{
+   "personId":"1",
```

Comment:
Rather returning "1" as `personId`, shall we return with path parameter customer is using to query?

Reply:
Fixed